core/os/os.cpp

```diff
@@ -333,6 +333,18 @@ void OS::set_has_server_feature_callback(HasServerFeatureCallback p_callback) {
 	has_server_feature_callback = p_callback;
 }
 
+String OS::get_name_simplified() const {
+	String simplified_name;
+	String name = get_name();
+	if (name == "Linux" || name == "FreeBSD" || name == "NetBSD" || name == "OpenBSD" || name == "BSD") {
+		simplified_name = "linuxbsd";
+	} else {
+		simplified_name = name.to_lower();
+	}
+
+	return simplified_name;
+}
```

Comment:
This is pretty hacky, and shouldn't here. The abstract `OS` class shouldn't know about its implementations so it shouldn't hardcode a special case.

You can do this logic in the editor code.